flexibleengine/acceptance/resource_flexibleengine_elb_security_policy_test.go

```diff
@@ -0,0 +1,120 @@
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getSecurityPoliciesV3ResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := OS_REGION_NAME
+	// getSecurityPolicy: Query the ELB security policy
+	var (
+		getSecurityPolicyHttpUrl = "v3/{project_id}/elb/security-policies/{security_policy_id}"
+		getSecurityPolicyProduct = "elb"
+	)
+	getSecurityPolicyClient, err := cfg.NewServiceClient(getSecurityPolicyProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating SecurityPolicies Client: %s", err)
+	}
+
+	getSecurityPolicyPath := getSecurityPolicyClient.Endpoint + getSecurityPolicyHttpUrl
+	getSecurityPolicyPath = strings.ReplaceAll(getSecurityPolicyPath, "{project_id}", getSecurityPolicyClient.ProjectID)
+	getSecurityPolicyPath = strings.ReplaceAll(getSecurityPolicyPath, "{security_policy_id}", fmt.Sprintf("%v", state.Primary.ID))
+
+	getSecurityPolicyOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+	}
+	getSecurityPolicyResp, err := getSecurityPolicyClient.Request("GET", getSecurityPolicyPath, &getSecurityPolicyOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving SecurityPolicies: %s", err)
+	}
+	return utils.FlattenResponse(getSecurityPolicyResp)
+}
+
+func TestAccSecurityPoliciesV3_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "flexibleengine_elb_security_policy.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getSecurityPoliciesV3ResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testSecurityPoliciesV3_basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "protocols.0", "TLSv1.1"),
+					resource.TestCheckResourceAttr(rName, "protocols.1", "TLSv1.2"),
+					resource.TestCheckResourceAttr(rName, "ciphers.0", "ECDHE-ECDSA-AES128-SHA"),
+					resource.TestCheckResourceAttr(rName, "ciphers.1", "ECDHE-RSA-AES256-SHA"),
+				),
+			},
+			{
+				Config: testSecurityPoliciesV3_basic_update(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "protocols.0", "TLSv1.2"),
+					resource.TestCheckResourceAttr(rName, "ciphers.0", "ECDHE-ECDSA-AES128-SHA"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testSecurityPoliciesV3_basic(name string) string {
+	return fmt.Sprintf(`
+resource "flexibleengine_elb_security_policy" "test" {
+  protocols = [
+    "TLSv1.1",
+    "TLSv1.2"
```

Comment:
advise to cover all dictionary values

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/db44f9a4-4e42-49e3-8c88-f9132622d91c)
ok, i have revised